packages/stateful/actions/components/Stake.tsx

```diff
@@ -265,7 +266,7 @@ export const StakeComponent: ActionComponent<StakeOptions, StakeData> = ({
 
         {/* Choose source validator. */}
         <ValidatorPicker
-          displayClassName="grow"
+          displayClassName="grow min-w-0"
```

Comment:
**question**: is this one of those weird styling things where it wants a min h/w of 0 so flex can function properly?

Reply:
Yeah sorta. Flex by default set its children `min-width` properties to `auto`, which means they will be as large as their content inside, overflowing the container instead of truncating text internally. Setting `min-width` to `0` means they will not overflow their containers and the text truncates! yay